src/Test/Utilities/Portable/Metadata/ILValidation.cs

```diff
@@ -303,8 +303,12 @@ public static unsafe string GetMethodIL(this ImmutableArray<byte> ilArray)
         public static Dictionary<int, string> GetSequencePointMarkers(string pdbXml, string source = null)
         {
             string[] lines = source?.Split(new[] { "\r\n" }, StringSplitOptions.None);
-            var doc = new XmlDocument();
-            doc.LoadXml(pdbXml);
+            var doc = new XmlDocument() { XmlResolver = null };
+            using (var sr = new StringReader(pdbXml))
+            using (var reader = new XmlTextReader(sr) { DtdProcessing = DtdProcessing.Prohibit })
```

Comment:
📝 `reader.Dispose()` already calls `sr.Dispose()`, so there is no need for nested `using` statements.

```suggestion
            using (var reader = new XmlTextReader(new StringReader(pdbXml)) { DtdProcessing = DtdProcessing.Prohibit })
```